package.json

```diff
@@ -56,7 +56,7 @@
     "express-handlebars": "^3.0.0",
     "express-session": "^1.15.4",
     "gulp-merge-json": "^1.1.0",
-    "helmet": "^3.8.0",
+    "helmet": "^3.8.1",
```

Comment:
I don't think these version updates are related to this PR - could they be separated?